jaxlib/mosaic/gpu/custom_call.cc

```diff
@@ -119,7 +220,61 @@ void EnsureLLVMNVPTXTargetIsRegistered() {
   });
 }
 
-absl::StatusOr<std::pair<std::string, std::string>> GetSmAndPtxIsaVersion() {
+absl::StatusOr<int> GetLatestPtxasPtxIsaVersion() {
+  std::vector<const char*> ptxas_args = {"ptxas", "--input-as-string",
+                                         ".version 99.99", nullptr};
+  auto result = RunCUDATool("ptxas", ptxas_args);
```

Comment:
Avoid using auto when the type isn't obvious.

https://github.com/openxla/xla/blob/0b1b9b9066573c8bd5b2768bccf9a79e7d30b6db/xla/tsl/platform/default/statusor.h#L22

Reply:
Thank you, fixed.